src/services/infrastructure/url-generator/url.generator.service.ts

```
@@ -264,16 +130,13 @@ export class UrlGeneratorService {
+        return await this.getForumDiscussionUrlPathByProfileID(profile.id);
       case ProfileType.INNOVATION_HUB:
-        const innovationHubEntityInfo = await this.getNameableEntityInfoOrFail(
-          'innovation_hub',
-          this.FIELD_PROFILE_ID,
-          profile.id
-        );
+        const innovationHubEntityInfo =
+          await this.getNameableEntityInfoForProfileOrFail(
+            'innovation_hub',
+            profile.id
+          );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)